rootfs/usr/local/bin/generate-kubecfg

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Something about this should tie it to being necessary for usage with `aws-iam-authenticator` (like a comment) =)

Reply:
The thought was that this would be the way to generate kubecfg for all usage (where folks don't want to `kops export kubecfg`) not necessarily just for the `aws-iam-authenticator` use case